### ISSUE TITLE
[CI] - Freemarker + Quarkus main

### ISSUE THREAD

**Issue by gastaldi** [quarkiverse/quarkiverse#7]
This issue will be open and closed dependent on the state of https://github.com/quarkiverse/quarkiverse-freemarker building against Quarkus master SNAPSHOT.

If you have interest in being notified of this, please subscribe to the issue.

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkiverse-freemarker/actions/runs/301475873

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkiverse-freemarker/actions/runs/319145802

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkiverse-freemarker/actions/runs/321329967

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkiverse-freemarker/actions/runs/323477012

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/573807017

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/577101512

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/3425508012

**Comment by vsevel** [comment]
hi @gastaldi 
I merged the quarkus `3.x` https://github.com/quarkiverse/quarkus-freemarker/pull/156 change on freemarker `main`.
hence I suppose the build fail on quarkus `main`
was this the right thing to do?

**Comment by gastaldi** [comment]
@vsevel not until the ecosystem CI workflow is updated to understand that. 

Pinging @maxandersen @ia3andy @aloubyansky too

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/3434156562

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/3442648985

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/3458697571

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/3467602238

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/3476542361

**Comment by gastaldi** [comment]
@vsevel I pushed https://github.com/quarkiverse/quarkus-freemarker/commit/a27efdc96c39c7369456a433d9bb017a816a9771 to the master branch, it should (hopefully) fix the build

**Comment by vsevel** [comment]
@gastaldi cool!
when `3.0` is merged into main, I only have to remove the `ALTERNATIVE` property?
or do we need to revert the other changes as well (e.g. `uses: dcarbone/... actions/...@v3`)
is this safe to apply your approach to the vault extension as well? or do you we want to keep this on freemarker only?

**Comment by gastaldi** [comment]
@vsevel I have improved the ecosystem CI script to not require any changes in the extension repositories. 

I have reverted the change that adds the `ALTERNATIVE` property but kept the version bumps.

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/3494325189

**Comment by vsevel** [comment]
Bravo @gastaldi

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4200854400

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4220222415

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4229443111

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4239587311

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4528868957

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4539453773

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4550436871

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4560778296

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4571746228

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4593078974

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4604160046

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4615108339

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/4625617546

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/5550660830

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/5571953725

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/5829089215

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/5852032575

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/5863782687

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/5874851943

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/5886962046

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/7565568807

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/7579676794

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10313925170

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10313968631

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10346178474

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10346222359

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10363899793

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10363944302

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10381712522

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10381761128

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10399065935

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10399105737

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10414931967

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10414972447

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10447465030

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10447514628

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10465580459

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10465631706

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10483817838

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10483862603

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10502130733

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10502175063

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10520171492

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10520215781

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10553868625

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10553915983

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10572005776

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10572054942

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10590505585

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10590551519

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10609013857

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10609063508

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10627122025

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10627167710

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10661075073

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10661123697

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10676847790

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10676893576

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10695414561

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10714235432

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10732737061

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10732780670

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10766864339

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10766915589

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/10786093571

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-oidc-proxy/actions/runs/10786140833

**Comment by vsevel** [comment]
hello @gastaldi 
this issue keeps reopening because of an issue on `quarkus-oidc-proxy`, then closing on a successful build.
is there a problem on https://github.com/quarkiverse/quarkus-oidc-proxy or `quarkus-freemarker` ?

**Comment by vsevel** [comment]
in the `quarkus-oidc-proxy` build I see: 
```
2024-09-10T06:33:13.4825920Z [ERROR]   mvn <args> -rf :quarkus-oidc-proxy
2024-09-10T06:33:13.5074499Z Tests failed
2024-09-10T06:33:13.5074978Z Attempting to report results
2024-09-10T06:33:14.0184731Z [jbang] Resolving dependencies...
2024-09-10T06:33:14.1089714Z [jbang]    org.kohsuke:github-api:1.308
2024-09-10T06:33:14.6933823Z [jbang]    info.picocli:picocli:4.2.0
2024-09-10T06:33:14.8068285Z [jbang] Dependencies resolved
2024-09-10T06:33:14.8761191Z [jbang] Building jar for quarkus-ecosystem-issue.java...
2024-09-10T06:33:15.9247190Z The CI build had status failure.
2024-09-10T06:33:17.3441893Z Report issue found: [CI] - Freemarker + Quarkus main - https://github.com/quarkiverse/quarkiverse/issues/7
2024-09-10T06:33:17.3443059Z The issue is currently CLOSED
2024-09-10T06:33:18.7429161Z Comment added on issue https://github.com/quarkiverse/quarkiverse/issues/7 - https://github.com/quarkiverse/quarkiverse/issues/7#issuecomment-2339784750, the issue has been re-opened
```

where is set the association between the repo and the issue?

**Comment by gastaldi** [comment]
That is really weird, sounds like a bug in the quarkus-ecosystem-issue.java, I'll investigate what's going on

**Comment by gastaldi** [comment]
@vsevel it was a bug in https://github.com/quarkiverse/quarkus-oidc-proxy/commit/8bb7973d901cf6ef5bfe5b17fea78079fde5d5fc, which is now fixed.

**Comment by vsevel** [comment]
gosh I cloned the repo, and searched for issue '7'. 
did not think about looking for `freemarker`.
thanks @gastaldi

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/12626842706

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/12645514974

**Comment by quarkiversebot** [comment]
Unfortunately, the build failed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/20609890318

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/20630628927

**Comment by quarkiversebot** [comment]
The build is still failing:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/20648905849

**Comment by quarkiversebot** [comment]
Build fixed:
* Link to latest CI run: https://github.com/quarkiverse/quarkus-freemarker/actions/runs/20702575073
